src/renderers/shared/fiber/ReactFiberContext.js

```diff
@@ -64,18 +64,22 @@ exports.hasContextChanged = function() : boolean {
   return index > -1 && didPerformWorkStack[index];
 };
 
-exports.isContextProvider = function(fiber : Fiber) : boolean {
+function isContextProvider(fiber : Fiber) : boolean {
   return (
     fiber.tag === ClassComponent &&
+    // Instance might be null, if the fiber errored during construction
+    fiber.stateNode &&
```

Comment:
Is this only true during unwinding? Can it be null in any other place in the code? We rely on `stateNode` existing on classes in a lot of places so I'm just trying to understand if something else could fail now.